### DISCUSSION TITLE
trim on checkbox/toggle values

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [quasarframework/quasar#12429] [General - Components / Directives / etc]
Is there a special reason why the click event is also emitted together with the value? I tried to use `.trim` on the modelvalue, but since there is no check if the `rawArgs` are all string trim fails on trying to call trim on a clickevent.

https://github.com/vuejs/core/blob/9c304bfe7942a20264235865b4bb5f6e53fdee0d/packages/runtime-core/src/componentEmits.ts#L121

https://github.com/quasarframework/quasar/blob/b32daffa22b547e887808e312ede55a30a247f05/ui/src/components/checkbox/use-checkbox.js#L150

**Comment by metalsadman** [comment]
some users requested for it, iirc.
